web/src/lib/components/forms/create-user-form.svelte

```diff
@@ -13,9 +14,17 @@
   let confirmPassowrd = '';
 
   let canCreateUser = false;
-
+  let quotaSize: number | undefined = undefined;
   let isCreatingUser = false;
 
+  $: warning = quotaSize && convertToBytes(Number(quotaSize), 'GiB') > $serverInfo.diskSizeRaw;
+
+  onMount(async () => {
+    if (!$serverInfo) {
+      const { data } = await api.serverInfoApi.getServerInfo();
+      $serverInfo = data;
+    }
+  });
```

Comment:
why is this needed? it seems like it should be initialized in the layout or when the user logs in, not in every component that uses it.

Reply:
It feels a bit overkill to have this API request in a layout. I moved it to the page.ts, can you check my last commit ?

Reply:
Why is it overkill to have it in the layout? That's essentially an entrypoint and means it will only be called once per _page load_.

Reply:
It may be more complicated since it requires auth, so we can look into this again in the future.